### PR TITLE
urler 0.1 (new formula)

### PR DESCRIPTION
**Pull request by marianposaceanu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

Small issue: the repo. is too [new](https://twitter.com/bagder/status/1641809063691386884).

```
urler:
  * GitHub repository too new (<30 days old)
Error: 1 problem in 1 formula detected
```

### CODE REVIEW

File: Formula/urler.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Urler < Formula
+  desc "Command-line tool for URL parsing and manipulation"
+  homepage "https://github.com/curl/urler"
+  url "https://github.com/curl/urler/archive/64a48c004cbc0e91d15aff2e090f423ab7555b6c.tar.gz"
+  version "0.1"
```
Comment (carlocab) [inline Formula/urler.rb:5]: We don't build from arbitrary commits. Please ask upstream if they'll consider tagging releases in their repository.

File: Formula/urler.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "730d4d2611d1844a0389b71a214bd8b0b1c86724f65ec913d777af482a7db496"
+  license "curl"
+
+  def install
+    system "make"
+    bin.install "urler"
+    man1.install "urler.1"
```
Comment (carlocab) [inline Formula/urler.rb:12]:
```suggestion
    system "make", "BINDIR=#{bin}", "MANDIR=#{man}", "install"
```

**Review by carlocab — DISMISSED** [review]

**Comment by ZhongRuoyu** [comment]
0.1 is not released yet. It is still in active development. Last commit was made 6 minutes ago: https://github.com/curl/urler.

File: Formula/urler.rb (hunk 2 above)
Comment (marianposaceanu) [reply Formula/urler.rb]: Created an upstream PR to add support for this https://github.com/curl/urler/pull/17

File: Formula/urler.rb (hunk 1 above)
Comment (marianposaceanu) [reply Formula/urler.rb:5]: Thanks as @ZhongRuoyu just commented version 0.1 is not released yet, this PR is a bit too green.

**Comment by ZhongRuoyu** [comment]
Sorry @marianposaceanu. trurl 0.2 (renamed from urler) was released and another contributor has opened #127447. Nevertheless, thanks for the PR!

**Pull request closed without merging**